src/react-uploader/list.js

```diff
@@ -14,7 +14,11 @@ export default class List extends React.Component {
 
           <div className="upl-list-img">
             <div className="img-center">
-              <img src={item.cdnUrl || item.url} />
+              {item.isImage && (item.cdnUrl || item.url) ?
+                <img src={item.cdnUrl || item.url} />
```

Comment:
instead of just saying preview not available can we hide the gray box and just show the filename?